app/policies/application_policy.rb

```diff
@@ -6,6 +6,84 @@ def initialize(user, record)
     @record = record
   end
 
+  # Checks if the primary person of the given family is an individual market primary family member.
+  # A primary person is considered an individual market primary family member if they have either a consumer role or a resident role,
+  # and the user is the same as the user of the primary person.
+  #
+  # @param family [Family] The family to check.
+  # @return [Boolean] Returns true if the primary person of the family is an individual market primary family member, false otherwise.
+  def individual_market_primary_family_member?(family)
```

Comment:
- We are initializing FamilyPolicy with family record. 
- Since record itself is family in application policy, do we have to pass family to this method?